### ISSUE TITLE
Event listeners don't appear to work if the listener and emitter reside on the same DOM element

### ISSUE THREAD

**Issue by brettburwell** [cmalven/modu#4]
For example. If you have the following element: 
```
<div class="fancy-pants"
     data-module-slider
     data-module-filter
>
```

And the `Filter` class has the following event emitter:
```
this.emit('filtersUpdated');
```

The following listener doesn't fire in the `Slider` class:
```
this.on('Filter', 'filtersUpdated', () => {});
```

If I change the element that `Filter` is initialized from everything works as expected. 
```
<div class="fancy-pants"
     data-module-slider
>
  <div class="fancy-pants__inner"
       data-module-filter
  >
```

**Comment by cmalven** [comment]
@brettburwell This is fixed in 1.5.2 https://github.com/cmalven/modu/releases/tag/1.5.2

**Comment by brettburwell** [comment]
Awesome! Thanks for the update.
